rbac/rbac/settings.py

```diff
@@ -289,11 +289,12 @@
 
 ACCESS_CACHE_DB = 1
 ACCESS_CACHE_LIFETIME = 10 * 60
+ACCESS_CACHE_ENABLED = ENVIRONMENT.bool("ACCESS_CACHE_ENABLED", default=True)
+ACCESS_CACHE_CONNECT_SIGNALS = ENVIRONMENT.bool("ACCESS_CACHE_CONNECT_SIGNALS", default=True)
```

Comment:
Yes. With `ACCESS_CACHE_ENABLED` set to `False`, then making calls to the `AccessCache` functions would be no-ops. With it set to `True` and the `ACCESS_CACHE_CONNECT_SIGNALS` set to False, you can still interact with the `AccessCache` and make changes to Redis, you just don't get the model signals firing.

Reply:
Just within the context of running the seeds though, wouldn't it be the same? For instance, with the changes in #300 we started running seeds like: `pipenv run sh -c "ACCESS_CACHE_ENABLED=0 python rbac/manage.py seeds"` and had the seeds clear the cache explicitly (no need for `AccessCache`) after running which gave us a considerable increase alone.

I suppose this could be more useful and explicit moving forward though provided we need to do that?

Reply:
Need? No. I see that the code is bypassing the implementation in `AccessCache` - thinking about it now, since `AccessCache` is meant to abstract away the fact that we're using Redis, I think my preference is to make the cache clearing done through the `AccessCache` implementation anyway. If that's the case, then it sounds like the two settings would make sense. Thoughts?

Reply:
Yeah, I definitely agree with that. 👍